### PR TITLE
Clarity and $ updates

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Clarity and $ updates 
- **Is this PR related to a Monolith release?** No

Netlify Preview Link: https://preview-4902--reverent-murdock-829d24.netlify.app/docs/guides/validate-access-tokens/dotnet/main/

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Faccesstoken.md) | [95](https://okta.acrolinx.cloud/api/v1/checking/scorecards/24de8303-c26b-4e79-8e3e-643eefd9ccad) | 100  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Fretrievekeys.md) | [91](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8aa9cea1-6861-4255-bbe2-cd9d30527f6f) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/go/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fgo%2Faccesstoken.md) | [93](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f22d9971-c1d8-47a4-a634-e46240383e40) | 100  | 66  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/go/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fgo%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/3534444e-6d2e-41e4-b1dd-970fb26b43ae) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Findex.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/00ddb212-89b7-40ac-be31-63cf5d270657) | 88  | 88  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/java/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fjava%2Faccesstoken.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d1a56222-6235-4a3a-8687-7ca2ea21874c) | 100  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/java/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fjava%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ab253a37-0b5e-4a15-9600-c0ce1ff4b967) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/nodejs/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fnodejs%2Faccesstoken.md) | [98](https://okta.acrolinx.cloud/api/v1/checking/scorecards/322e5ec6-1a79-49e9-9755-4d0a10d6174a) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/nodejs/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fnodejs%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f1a70d08-e4cd-4ddb-834a-f5e5dda553ce) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/python/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fpython%2Faccesstoken.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/209a1940-b4af-4ecd-9ad3-1c1e6c02d9db) | 100  | 67  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/python/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/cb82ede6f46e0808e09d240f3d8c0df48758f92c/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fpython%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/dec18157-3ae1-4839-ad0b-52ff1db85741) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.7b594864-4b4b-489e-9dc2-59d8c0d55866?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md [hunk 1]
```diff
@@ -117,9 +99,9 @@ if (validatedToken.Header?.Alg == null || validatedToken.Header?.Alg != expected
 }
 ```
 
-### Additional validation for ID tokens
+### More validation for ID tokens
 
-When validating an ID token, you should verify that the `aud` (Audience) claim equals the Client ID of the current application.
+When validating an ID token, you should verify that the `aud` (audience) claim equals the client ID of the current application.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md:104]:
```suggestion
When validating an ID token, you should verify that the `aud` (audience) claim equals the client ID of the current app.
```

File: packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md [hunk 2]
```diff
@@ -1,74 +1,72 @@
...
+* Verify the token signature.
+* Verify the claims.
 
-<StackSelector snippet="accesstoken" noSelector/>
+<StackSnippet snippet="accesstoken" />
 
 ## Validate a token remotely with Okta
 
-Alternatively, you can also validate an access or refresh token using the Token Introspection endpoint: [Introspection Request](/docs/reference/api/oidc/#introspect). This endpoint takes your token as a URL query parameter and returns back a simple JSON response with a boolean `active` property.
+Alternatively, you can also validate an access or refresh token using the Token Introspection endpoint: [Introspection request](/docs/reference/api/oidc/#introspect). This endpoint takes your token as a URL query parameter and returns a simple JSON response with a boolean `active` property.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md:64]:
```suggestion
Alternatively, you can also validate an access or refresh token using the Token Introspection endpoint: [Introspection request](/docs/reference/api/oidc/#introspect). This endpoint takes your token as a URL query parameter and returns a simple JSON response with a Boolean `active` property.
```

**Review by barbaravo-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Faccesstoken.md) | [95](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2913e9c0-7887-493a-ba4c-48ebae3ab6ed) | 100  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Fretrievekeys.md) | [91](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6ad2aff1-540c-47b2-bb13-a8bed7396a73) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/go/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fgo%2Faccesstoken.md) | [93](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2bd3d987-8dd8-4d1d-8288-be902d2de440) | 100  | 66  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/go/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fgo%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0b1e06af-b7f1-43a5-941a-eb792a20ecf6) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Findex.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/11dded04-cbcd-46d0-b101-4ffb0708a53f) | 88  | 88  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/java/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fjava%2Faccesstoken.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d8b59b31-4794-4415-804e-9824c9b5e102) | 100  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/java/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fjava%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f3700268-e3d2-4938-90b1-2ae88808a59d) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/nodejs/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fnodejs%2Faccesstoken.md) | [98](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6b7b8b44-e9ab-4390-91a4-71e7df054674) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/nodejs/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fnodejs%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f4576672-5993-4e37-a86a-b5d864e9970a) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/python/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fpython%2Faccesstoken.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/36462b75-183b-4c6d-8464-41e2b02f0a24) | 100  | 67  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/python/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/a6cb1ee2592e1f175a31ba3f277e2f462d7f0201/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fpython%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/44041bf9-66f9-4fbf-bfbe-318f3cab3ef7) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b3fc578a-0de6-4577-8698-87d2d90fcfd1?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **30** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:30** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Faccesstoken.md) | [95](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94129ad5-7bb7-40e9-8841-5701ac22636c) | 100  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/dotnet/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fdotnet%2Fretrievekeys.md) | [91](https://okta.acrolinx.cloud/api/v1/checking/scorecards/86705121-d537-4d28-ba3a-c8c6b5c43670) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/go/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fgo%2Faccesstoken.md) | [93](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2e66ef94-4998-48f5-b2a7-5afe257fa485) | 100  | 66  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/go/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fgo%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8bfd9e02-a7b4-4cf6-ae75-e87bbc8b71c8) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/index.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Findex.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1b8568dd-fe58-443a-ad50-41edc29ff7ea) | 88  | 88  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/java/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fjava%2Faccesstoken.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ecfad4ab-055d-4804-b989-00aebb18aa4b) | 100  | 71  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/java/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fjava%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/7e8c547a-c483-48cc-86bf-9db42cf5356f) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/nodejs/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fnodejs%2Faccesstoken.md) | [98](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a1627b8a-f651-4db8-9d75-2622a942711d) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/nodejs/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fnodejs%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/738f36f6-2764-482e-86ce-a530cd754cc6) | 100  | 100  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/python/accesstoken.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fpython%2Faccesstoken.md) | [86](https://okta.acrolinx.cloud/api/v1/checking/scorecards/debfbde6-a92c-4fc8-9ffa-9d7070885600) | 100  | 67  | 100  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/validate-access-tokens/main/python/retrievekeys.md](https://github.com/okta/okta-developer-docs/blob/939b2849363cd529fcd8862a427b0b9f02e5c16d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fvalidate-access-tokens%2Fmain%2Fpython%2Fretrievekeys.md) | [89](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2ea8b766-8f3a-4458-b370-30813a3321cb) | 100  | 100  | 100  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4a03e2eb-a9de-4b13-a7d0-963c518adb87?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**